packages/orbit-components/src/Wizard/Wizard.stories.tsx

```diff
@@ -88,7 +87,7 @@ export const Playground: Story = {
   render: ({ activeStep, completedSteps, labelProgress, steps, ...args }) => (
     <Wizard
       {...args}
-      labelProgress={`${completedSteps} ${labelProgress} ${steps.length}`}
+      labelProgress={`${completedSteps + 1} ${labelProgress} ${steps.length}`}
```

Comment:
I can get to this situation, is it expected?

<img width="158" alt="image" src="https://github.com/user-attachments/assets/c3b6cd19-2409-49b4-8c90-c1b89f74a0b1" />

Reply:
🤔 I don't think so. I'll fix

Reply:
It's weird because what does "X of 7" means? Does it mean I'm on step X? Or that I completed X steps (and so I'm on step X+1)? It's weird… I guess I won't change anything after all